app/build.gradle

```diff
@@ -289,7 +289,7 @@ task printGeckoviewVersion {
         def configuration = configurations.implementationDependenciesMetadata
         def dependencies = configuration.incoming.resolutionResult.allDependencies
         def geckoviewDependency = dependencies.find { it.selected.id.moduleIdentifier.group == 'org.mozilla.geckoview' }
-        println('geckoviewVersion: ' + geckoviewDependency.selected.moduleVersion.version)
+        println('geckoviewVersion: ' + groovy.json.JsonOutput.toJson(geckoviewDependency.selected.moduleVersion.version))
```

Comment:
Encoding it a json allows uniformly handling this output by removing the prefix and parsing it as json.

Reply:
Oh, you're referring to how you can [uniformly parse any `gradle` output](https://github.com/mozilla-mobile/reference-browser/pull/737/files#diff-b21033916484673312ae5ae0cf5c9e8cR38).

We still need to handle/parse each `gradle` command individually, though - each `gradle` command's output is used differently.
Arguably, we're saving the extra call to `json.load(...)` for those `gradle` commands that need it, but we're just moving that to the other side: now we need to add `toJson(...)` for all the commands that don't need to be JSON by default - plus, now `./gradlew printGeckoviewVersion` is less usable, since it has unnecessary extra quotes.